rovercode/app.py

```diff
@@ -28,6 +29,7 @@
 CHAINABLE_RGB_MANAGER = None
 CLIENT_ID = None
 SESSION = None
+GROVEPI_QUEUE = []
```

Comment:
You might want to use the [queue](https://docs.python.org/3/library/queue.html) from the Python standard library